src/empty-placeholder.ts

```diff
@@ -0,0 +1,81 @@
+import { css, html, LitElement, CSSResultGroup, nothing } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { choose } from 'lit/directives/choose.js';
+
+import emptyQueryIcon from './assets/img/icons/empty-query';
+import nullResultIcon from './assets/img/icons/null-result';
+
+export type PlaceholderType = 'empty-query' | 'null-result' | null;
+@customElement('empty-placeholder')
+export class EmptyPlaceholder extends LitElement {
+  @property({ type: String }) placeholderType: PlaceholderType = null;
+
+  @property({ type: Boolean }) isMobileView?: false;
+
+  render() {
+    return this.placeholderType
+      ? html`${this.getPlaceholderTemplate}`
```

Comment:
One small thing: please change `getPlaceholderTemplate` to `placeholderTemplate `